collector/shared/version.go

```diff
@@ -1,42 +0,0 @@
-package collector_shared
```

Comment:
I dont know about this is looks like  version checking functions are broken which will be needed to say if 3.0 do X , if 3.2 do y , if 3.4  do z right